Dockerfile

```diff
@@ -52,4 +52,4 @@ ENV FLUENTD_CONF="fluent.conf"
 EXPOSE 24224
 
 ### docker run -p 24224 -v `pwd`/log: -v `pwd`/log:/home/ubuntu/log fluent/fluentd:latest
-CMD fluentd -c /fluentd/etc/$FLUENTD_CONF -p /fluentd/plugins $FLUENTD_OPT
+CMD exec fluentd -c /fluentd/etc/$FLUENTD_CONF -p /fluentd/plugins $FLUENTD_OPT
```

Comment:
From docker document, @c0b 's way is recommended.

https://docs.docker.com/reference/builder/#cmd

@cpuguy83 Could you update the patch?